az-snp-vtpm/src/vtpm.rs

```diff
@@ -24,6 +24,7 @@ use tss_esapi::Context;
 
 const VTPM_HCL_REPORT_NV_INDEX: u32 = 0x01400001;
 const VTPM_AK_HANDLE: u32 = 0x81000003;
+const VTPM_QUOTE_PCR_SLOTS: [PcrSlot; 3] = [PcrSlot::Slot15, PcrSlot::Slot16, PcrSlot::Slot22];
```

Comment:
since we don't provide a way to specify these, let's atleast include all the ones that we know are currently in use:
`0-7+14`. That way they are attested and we can later decide whether to check the values or not.

Reply:
👍 I changed the PCR set and tested it.